velox/functions/prestosql/tests/ArrayNGramsTest.cpp

```diff
@@ -47,26 +47,23 @@ TEST_F(ArrayNGramsTest, integers) {
   testNgram<int64_t>({1, 2, 3, 4}, 4, {{{1, 2, 3, 4}}});
   testNgram<int64_t>({1, 2, 3, 4}, 5, {{{1, 2, 3, 4}}});
   testNgram<int64_t>(
-      {1, 2, 3, 4}, std::numeric_limits<int32_t>::max(), {{{1, 2, 3, 4}}});
-  testNgram<int64_t>(
-      {1, 2, 3, 4},
-      std::numeric_limits<int32_t>::max() + (int64_t)(1000),
-      {{{1, 2, 3, 4}}});
+      {1, 2, 3, 4}, std::numeric_limits<int>::max(), {{{1, 2, 3, 4}}});
```

Comment:
Why change std::numeric_limits<int32_t> to std::numeric_limits<int> here?

Reply:
yea, [here is the presto implementation](https://github.com/prestodb/presto/blob/4b458d41b8988f02c09bf9ff85e5d7bacf093b98/presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayNgramsFunction.java#L39)

Reply:
> Why change std::numeric_limits<int32_t> to std::numeric_limits here?

Sorry not intentional. Fixed.